topics/bloomfilters.md

```diff
@@ -134,6 +212,47 @@ Example of default bloom filter information:
 16) (integer) 26214300
 ```
 
+### When to adjust default configurations
+
+Adjusting the default configurations below can be used to update the behavior of the default bloom object created and are beneficial in different scenarios:
+
+- **Higher capacity (bf.bloom-capacity)**: Increase this value when you expect to store a larger number of items in all your new bloom filter objects. It increases the overall capacity for non scaling filters. For scaling filters, it increases the starting/initial capacity which reduces the need for additional scaling operations which can improve performance.
+
+- **Lower false positive rate (bf.bloom-fp-rate)**: Decrease this value when correctness is critical for your application and you want to reduce the number of false positives in all your new bloom filter objects. For example, in fraud detection or security applications where false positives are costly.
+
+- **Higher expansion rate (bf.bloom-expansion)**: Increase this value when you want faster growth of bloom filters that will scale across all your new scaling bloom filter objects. This reduces the number of scaling operations but uses more memory quicker.
+
+- **Lower tightening ratio (bf.bloom-tightening-ratio)**: Lower this when you want to maintain a more consistent false positive rate across multiple scaling operations for all your new bloom filter objects. (Not advisable to change this default, as it is already set to a strict value of 0.5)
+
+- **Fixed seed (bf.bloom-use-random-seed)**: Set to false only when you want the new bloom filter object creations to use a fixed seed for deterministic occurrence of false positives during item add/exists operations. It can be used  for testing or reproducibility.
+
+You can modify these default values using the CONFIG SET command. However, note that the effect of the configuration change is only applicable to the bloom objects created after the configuration change where the property is not specified through the command already.  
+
+Example usage of changing all the different properties:
+```bash
+127.0.0.1:6379> CONFIG SET bf.bloom-fp-rate 0.001
+127.0.0.1:6379> CONFIG SET bf.bloom-capacity 1000
+127.0.0.1:6379> CONFIG SET bf.bloom-expansion 4
+127.0.0.1:6379> CONFIG SET bf.bloom-tightening-ratio 0.6
+127.0.0.1:6379> CONFIG SET bf.bloom-use-random-seed false
+```
+
+### Memory usage limit
+
+The `bf.bloom-memory-usage-limit` configuration (default 128MB) controls the maximum memory that a single bloom filter can use:
+
+Example usage of increasing the limit:
+```bash
+127.0.0.1:6379> CONFIG SET bf.bloom-memory-usage-limit 268435456
+```
+
+Having a limit on the max memory per bloom object prevents them from growing unbounded, thus maintaining server performance during serialization.
+If your use case requires bloom filters beyond this limit, you can increase this configuration value.
+
+However, be aware it can impact overall system performance and memory availability for other operations.
```

Comment:
```suggestion
If your use case requires bloom filters with capacity beyond what this limit supports, you can increase this configuration value. However, be aware it can impact overall system performance and memory availability for other operations.
```